swp/models/publication.py

```diff
@@ -42,6 +42,7 @@ class Publication(models.Model):
     abstract = models.TextField(_('abstract'), blank=True)  # [AB]
     authors = ArrayField(models.CharField(max_length=255), blank=True, null=True, verbose_name=_('authors'))  # [AU]
     publication_date = models.CharField(_('publication date'), max_length=255, blank=True, default='')  # [PY]
+    publication_dt = models.DateField(_('publication dt'), null=True, blank=True)
```

Comment:
This naming scheme is a bit confusing. It would be better to make clear which field contains which data just by the name. I would recommend:

- renaming the old `publication_date` CharField to `publication_date_raw`
- `publication_date` for your real DateField